app/views/site/keyboardMoveHelp.scala

```diff
@@ -103,6 +103,7 @@ object helpModal {
         tbody(
           header(performAMove()),
           row(kbd("e2e4"), movePieceFromE2ToE4()),
+          row(kbd("4244"), movePieceFromD2ToD4()),
```

Comment:
why a new translation when we could reuse the one just above `movePieceFromE2ToE4`